lib/gen_fsm.ex

```diff
@@ -20,69 +85,82 @@ defmodule GenFSM do
     end
   end
 
+  @spec stop(server, reason :: term, timeout) :: :ok
   def stop(fsm, reason \\ :normal, timeout \\ :infinity) do
     :gen.stop(fsm, reason, timeout)
   end
 
+  @type fsm_ref :: pid | name
```

Comment:
I am actually kind confused about that

Server is defined as `@type server :: pid | name | {atom, node}` and fsm_ref is defined to be `@type fsm_ref :: pid | name`. I don't know if it allows for `{atom, node}` the places it has been set to be an `fsm_ref`.